lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -90,7 +98,15 @@ class << self
           #         should be stored in a file list file.
           #
           def input_output_paths_use_filelist?(object)
-            object.project.object_version.to_i >= 50
+            unless object.project.root_object.compatibility_version.nil?
+              version_match = object.project.root_object.compatibility_version.match(/Xcode ([0-9]*.[0-9]*)/)
+              compatibility_version = version_match[1].to_f unless version_match.nil? || version_match.length < 2
```

Comment:
I think we also don't want `to_f` here - ex. `'10.10'` should keep the `.10` suffix and not result in `.1`

Reply:
of course I hope we never have 10+ minor updates to a major Xcode release.. but you never know :)

Reply:
Thanks for the catch, updated.